ci/management/commands/generate_reports.py

```diff
@@ -141,14 +152,21 @@ def get_identity(self, ids_client, identity):
         self.identity_cache[identity] = identity_object
         return identity_object
 
+    def get_registrations(self, hub_client, **kwargs):
+        registrations = hub_client.get_registrations(kwargs)
+        cursor = registrations['next']
+        while cursor:
+            for result in registrations['results']:
+                yield result
+            params = parse_cursor_params(cursor)
+            registrations = hub_client.get_registrations(params)
+            cursor = registrations['next']
+        for result in registrations['results']:
+            yield result
```

Comment:
I'm sure there's a nicer way of doing this but I'm leaving that investigation for when I start implementing this in the `seed-services-client` code base.